### PR TITLE
NXS-6194: Add IP trace API endpoint to SDK

### PR DESCRIPTION
**Pull request by mohsinr**
In this PR we add Customer Trace functionality so for any customer with customer code we can get trace and then IP and geo location etc.

### CODE REVIEW

File: Nexus.Sdk.Token/TokenServerProvider.cs [hunk 1]
```diff
@@ -610,6 +610,24 @@ public async Task<CustomerDataResponse> GetCustomerData(string customerCode)
             return await builder.ExecuteGet<CustomerDataResponse>();
         }
 
+        /// <summary>
+        /// Get customer trace based on the code
```
Comment (WJRobert) [inline Nexus.Sdk.Token/TokenServerProvider.cs:614]: Please update documentation/comments to map what is being done in method

**Review by WJRobert — CHANGES_REQUESTED** [review]

File: Nexus.Sdk.Shared/Facades/CustomersFacade.cs [hunk 2]
```diff
@@ -66,4 +66,16 @@ public async Task<DeleteCustomerResponse> Delete(DeleteCustomerRequest request,
     {
         return await _provider.DeleteCustomer(request, customerIPAddress);
     }
+
+    /// <summary>
+    /// Get customer trace based on the code
```
[the remark this answers is not in the document]
Comment (mohsinr) [reply Nexus.Sdk.Shared/Facades/CustomersFacade.cs:71]: [Update doc comments for Customer Trace endpoint](https://github.com/QuantozTechnology/Nexus.Sdk.Dotnet/pull/46/commits/5028eba0005f38889bce8839fdca5c3094e85b10)

File: Nexus.Sdk.Shared/Facades/Interfaces/ICustomersFacade.cs [hunk 3]
```diff
@@ -56,4 +56,13 @@ public interface ICustomersFacade
     /// Customer personal data
     /// </returns>
     public Task<CustomerDataResponse> GetData(string customerCode);
+
+    /// <summary>
```
[the remark this answers is not in the document]
Comment (mohsinr) [reply Nexus.Sdk.Shared/Facades/Interfaces/ICustomersFacade.cs:60]: [Update doc comments for Customer Trace endpoint](https://github.com/QuantozTechnology/Nexus.Sdk.Dotnet/pull/46/commits/5028eba0005f38889bce8839fdca5c3094e85b10)

File: Nexus.Sdk.Token/TokenServerProvider.cs (hunk 1 above)
Comment (mohsinr) [reply Nexus.Sdk.Token/TokenServerProvider.cs:614]: [Update doc comments for Customer Trace endpoint](https://github.com/QuantozTechnology/Nexus.Sdk.Dotnet/pull/46/commits/5028eba0005f38889bce8839fdca5c3094e85b10)

File: Nexus.Sdk.Shared/Facades/CustomersFacade.cs (hunk 2 above)
[the remark this answers is not in the document]
Comment (WJRobert) [reply Nexus.Sdk.Shared/Facades/CustomersFacade.cs:71]: Missed the `List customer traces ` part :) Please update this to all three places

**Review by WJRobert — CHANGES_REQUESTED** [review]

File: Nexus.Sdk.Shared/Facades/CustomersFacade.cs (hunk 2 above)
[the remark this answers is not in the document]
Comment (mohsinr) [reply Nexus.Sdk.Shared/Facades/CustomersFacade.cs]: Got it, it is updated thank you!

**Review by WJRobert — APPROVED** [review]

**Pull request merged**